2200049/ 문제05.py

```diff
@@ -0,0 +1,12 @@
+T = int(input())
+for i in range(T):
+  sum = 0
+  arr = list(map(int,input().split()))
+  for idx in range(len(arr)):
+    if (idx+1)%2==1:
+      sum+=arr[idx]*2
+    else:
+      sum+=arr[idx]
+  for N in range(0,10):
```

Comment:
좋은 풀이네요! 👍 
```suggestion
  for N in range(10):
```
`range`는 기본값으로 0부터 시작하기 때문에 생략 가능합니다 😄